shapely/tests/geometry/test_format.py

```diff
@@ -93,7 +93,12 @@ def test_format_polygon():
     )
 
     # 'g' format varies depending on GEOS version
-    if geos_version < (3, 10, 0):
+    if geos_version >= (3, 13, 0):
+        expected_2G = (
+            "POLYGON ((10 0, 7.07 -7.07, 6.12E-16 -10, -7.07 -7.07, "
+            "-10 -1.22E-15, -7.07 7.07, -1.84E-15 10, 7.07 7.07, 10 0))"
```

Comment:
FYI, I'm a bit uneasy (and pedantic) about this one an and considering an upstream change to zap the near-zero values to zero, so this test might change once more before GEOS 3.13 is finalized.

Reply:
xref https://github.com/libgeos/geos/pull/978